src/components/InstrumentPicker/InstrumentListItem.tsx

```diff
@@ -17,8 +17,10 @@ interface InstrumentListItemProps extends InstrumentPickerActions {
 const StyledInstrumentItem = styled(Flex)`
   margin-top: 10px;
   justify-content: space-between;
-  &.inactive:hover {
+  &:hover {
```

Comment:
should be blue and not only on hover, like in the trades/orders/orderbook widgets